hooks/pre-checkout

```diff
@@ -57,7 +57,9 @@ aws_strategy() {
       echo "Fetching secret from AWS secrets manager"
       ssh_key="$(aws secretsmanager get-secret-value --secret-id "${secret_key}" --region "${region}" | jq -r '.SecretString' | base64 --decode)"
 
-      ssh_key_file="${SECRETS_DIR}"/"${secret_key//\//\-}" # replace "/" in secret_key with "-"
+      # shellcheck disable=SC2001
+      clean_secret_key=$(echo "$secret_key" | sed 's#/#-#g') # replace "/" in secret_key with "-"
```

Comment:
not sure what we did during the last release. If we have used the exact same thing, then it is fine. But if this is new, we should probably try to run this snippet once on a macOS machine because it contains a different flavor of sed compared to nix systems.